com.unity.render-pipelines.core/Runtime/Lighting/ProbeVolume/ProbeBrickIndex.cs

```diff
@@ -75,7 +69,7 @@ struct BrickMeta
         internal ProbeBrickIndex(Vector3Int indexDimensions)
         {
             Profiler.BeginSample("Create ProbeBrickIndex");
-            int index_size = indexDimensions.x * (indexDimensions.y + 1) * indexDimensions.z;
+            int index_size = indexDimensions.x * indexDimensions.y * indexDimensions.z;
```

Comment:
Need to also update same calculation in InitProbeReferenceVolume (removing the +1)

Reply:
Also same in Clear here (can't link for whatever reason, but in the diff is line 125 of old and 111 of new.